browser-test/src/file.test.ts

```diff
@@ -92,6 +93,16 @@ describe('file upload applicant flow', () => {
       const error = await pageObject.$('.cf-fileupload-error')
       expect(await error?.isHidden()).toEqual(false)
     })
+
+    // TODO(#2988) Enable test once a11y issues are fixed.
+    // it('has no accessiblity violations', async () => {
```

Comment:
Non-blocking: Rather than commenting this out, you might try the [it.skip](https://jestjs.io/docs/api#testskipname-fn) method. It'll show that it was skipped in the test output and maybe make it easier to remember to come back and enabled later.

Reply:
Didn't know this existed. This is perfect, thanks!!